static_src/test/unit/stores/app_store.spec.js

```diff
@@ -262,12 +262,19 @@ describe('AppStore', function() {
   });
 
   describe('on app restarted', function() {
-    it('should emit a change', function() {
-      const spy = sandbox.spy(AppStore, 'emitChange');
+    it('should poll until running instances is greater then 0', function() {
```

Comment:
Hmm, the comment about firing an action on resolve was referring to this line, https://github.com/18F/cg-dashboard/pull/729/files#diff-d45ddca2d392b7558d8957b40dd036ccR242 I'm not sure how it got up here. anyway, sounds good to me.